src/charm.py

```diff
@@ -226,6 +228,25 @@ def main(self, event):
 
         self.model.unit.status = ActiveStatus()
 
+    def log_owasp_events(self, event):
+        owasp_logger = OWASPLogger(appid="dex.owasp-logger")
+        self.state.set_default(last_static_password="")
+
+        if not isinstance(event, ConfigChangedEvent):
+            self.logger.info("Not a config changed-event. Won't log any OWASP event.")
```

Comment:
Is this log info needed ?

Reply:
in this case I'd change it to `debug` type to avoid adding noise to the logs

Reply:
@NohaIhab how would we toggle in a running charm to show the `debug` logs?

Reply:
@kimwnasptd by setting the [`logging-config` model configuration](https://documentation.ubuntu.com/juju/3.6/reference/configuration/list-of-model-configuration-keys/#logging-config) for the units, this should be sufficient:
```
juju model-config logging-config="unit=DEBUG"
```